### PR TITLE
increase a bunch of timeouts...

### PR DESCRIPTION
**Pull request by jgrillo-grapl**
### Which issue does this PR correspond to?
NA

### What changes does this PR make to Grapl? Why?
This PR attempts to fix the integration tests by increasing a bunch of timeouts. So far it's not working. The e2e test is failing locally with 
```
---- test_sysmon_log_e2e stdout ----
Error: ErrorStatus(Status { code: Internal, message: "Timeout expired" })
thread 'test_sysmon_log_e2e' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`: the test returned a termination value with a non-zero status code (1) which indicates a failure', /rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/test/src/lib.rs:185:5
stack backtrace:
   0: rust_begin_unwind
             at ./rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at ./rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/core/src/panicking.rs:142:14
   2: core::panicking::assert_failed_inner
   3: core::panicking::assert_failed
             at ./rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/core/src/panicking.rs:181:5
   4: test::assert_test_result
             at ./rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/test/src/lib.rs:185:5
   5: sysmon_log_e2e_test::test_sysmon_log_e2e::{{closure}}
             at ./grapl/rust/e2e-tests/tests/sysmon_log_e2e_test.rs:42:1
   6: core::ops::function::FnOnce::call_once
             at ./rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/core/src/ops/function.rs:248:5
   7: core::ops::function::FnOnce::call_once
             at ./rustc/e092d0b6b43f2de967af0887873151bb1c0b18d3/library/core/src/ops/function.rs:248:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.


failures:
    test_sysmon_log_e2e

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 61.79s
```

### CODE REVIEW

**Review by colin-grapl — APPROVED** [review]
Approved, but I'd suggest also increasing the error threshold to something like 95% in tests.

**Comment by jgrillo-grapl** [comment]
> Approved, but I'd suggest also increasing the error threshold to something like 95% in tests.

What is the error threshold?

File: src/rust/rust-proto/src/client_macros.rs [hunk 1]
```diff
@@ -32,10 +32,10 @@ macro_rules! execute_client_rpc {
         $opts: expr,
     ) => {{
         {
-            let backoff = client_executor::strategy::FibonacciBackoff::from_millis(100)
+            let backoff = client_executor::strategy::FibonacciBackoff::from_millis(10)
                 .max_delay(Duration::from_millis(5000))
                 .map(client_executor::strategy::jitter);
-            let num_retries = 10;
+            let num_retries = 1000;
```
Comment (wimax-grapl) [inline src/rust/rust-proto/src/client_macros.rs:38]: 1000 retries feels wildly incorrect, especially when after fib(20)*10 of them each retry will delay up to 5sec long.

If this keeps retrying we'd just find outselves waiting for ~5sec * 980

**Review by wimax-grapl — CHANGES_REQUESTED** [review]

File: src/rust/rust-proto/src/client_macros.rs (hunk 1 above)
Comment (colin-grapl) [reply src/rust/rust-proto/src/client_macros.rs:38]: Agree, if an RPC is failing for even a matter of minutes I think we're beyond "retry" and we're in "a dependency is failing, we are no longer healthy".

**Review by colin-grapl — CHANGES_REQUESTED** [review]
Let's keep num_retries at 100, I don't think we want to block on RPCs for hours.

File: src/rust/rust-proto/src/client_macros.rs (hunk 1 above)
Comment (jgrillo-grapl) [reply src/rust/rust-proto/src/client_macros.rs:38]: Yes, I agree as well. This was an attempt to eliminate this variable as a possible cause for the observed failure.

**Comment by wimax-grapl** [comment]
even then colin it'd be ~= 5sec * 80 or 6 minutes? for a single RPC?

**Comment by jgrillo-grapl** [comment]
Closing this because it didn't help to isolate the error

**Pull request closed without merging**